lib/pthreads/examples/jointask_example.nit

```diff
@@ -0,0 +1,47 @@
+# This file is part of NIT (http://www.nitlanguage.org).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	 http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Simple example using threadpool
+module jointask_example
+
+import threadpool
+
+# Task computing a string
```

Comment:
As this is an example... please rework it a bit so it is a better introduction to the new service.

This doc is off, it does not compute a string, but it should!

And the following lines would benefit from more spacing.